plugins/jobtype/jobtypes/gobblin/private.properties

```diff
@@ -0,0 +1,7 @@
+jobtype.class=azkaban.jobtype.connectors.gobblin.GobblinHadoopJob
+
+jobtype.classpath=${plugin.dir}/lib/*
+gobblin.config.preset.dir=${plugin.dir}/presets
+
+#Gobblin file system URI TODO Can we make this cluster independent?
+fs.uri=hdfs://localhost:9000
```

Comment:
I was thinking the former to have localhost as default. I will put cluster specific setting in our deployment package.